src/templates/html/application/ApplicationHTML.tsx

```diff
@@ -23,16 +21,22 @@ function Highlights(props: { data: Application }): JSX.Element {
   const sessionId = appData.metadata.id;
 
   const appFee = appData.data.application.fee;
-  const feeCarrying = "payable" in appFee;
 
+  // assume no payment to start
+  let feeCarrying = false;
   let payRef: string | undefined = undefined;
   let feePaid: number = 0;
 
-  if (feeCarrying) {
+  // if this has a payment, set values
+  if (appFee && "payable" in appFee) {
+    feeCarrying = true;
     payRef = appFee.reference?.govPay;
-    feePaid = appFee.calculated * 100;
+    feePaid = appFee.calculated;
```

Comment:
`appFee.payable` here I think ? But correct that we no longer have to `* 100` :heavy_plus_sign: